single-line-diagram-force-layout/src/main/java/com/powsybl/sld/force/layout/Spring.java

```diff
@@ -0,0 +1,33 @@
+package com.powsybl.sld.force.layout;
+
+import org.jgrapht.graph.DefaultEdge;
+
+public class Spring extends DefaultEdge {
```

Comment:
As said earlier, I'm not sure we really need this class. The length and stiffness parameters could be directly set in the force layout algorithm, as they're here the same for all edges. Even if it could be interesting to regroup some nodes together by setting distrinct stiffness and lengths to edges according to some parameters.